iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java

```diff
@@ -107,17 +115,17 @@ public void testSendEmails() throws MessagingException {
     RegistrationRequestDto reg = createRegistrationRequest(username);
     notificationService.sendPendingNotifications();
 
-    Assert.assertEquals(1, wiser.getMessages().size());
+    assertEquals(1, wiser.getMessages().size());
```

Comment:
same holds for other checks, always try to use assertThat and hamcrest matchers that provide clearer error messages